api/FocusEvent.json

```diff
@@ -24,22 +24,22 @@
             "version_added": "9"
           },
           "opera": {
-            "version_added": true
+            "version_added": "15"
```

Comment:
https://staging-dot-mdn-bcd-collector.appspot.com/tests/api/FocusEvent confirmed not working in Opera 12.16.